src/panels/config/storage/dialog-mount-view.ts

```diff
@@ -114,13 +129,25 @@ class ViewMountDialog extends LitElement {
 
   @state() private _existing?: boolean;
 
+  @state() private _defaultBackupMount?: string | null;
+
   @state() private _reloadMounts?: () => void;
 
   public async showDialog(
     dialogParams: MountViewDialogParams
   ): Promise<Promise<void>> {
-    this._data = dialogParams.mount;
+    this._data =
+      dialogParams.mount !== undefined
+        ? {
+            ...dialogParams.mount,
+            default_backup_mount:
+              dialogParams.defaultBackupMount !== null
+                ? dialogParams.mount.name === dialogParams.defaultBackupMount
+                : undefined,
+          }
+        : undefined;
```

Comment:
Can we do this logic in a memoize function to get the data in render? So we don't have to add and delete this everywhere and use `_defaultBackupMount` as source of truth?

Reply:
Does it matter? We only run this one time when you open the dialog anyway?